source/_docs/pantheon-yml.md

```diff
@@ -1,12 +1,53 @@
 ---
-title: The pantheon.yml Configuration File
-description: Set site configuration in a pantheon.yml file stored in the root of their code repository.  
+title: Pantheon YAML Configuration Files
+description: Learn how to use a site-local configuration file to manage advanced settings at the site level and how to set default configurations across all sites running a Custom Upstream.
 tags: [pantheonyml, infrastructure]
 categories: []
 ---
+Hook into platform workflows and advanced site configurations using Pantheon's YAML configuration files, `pantheon.upstream.yml` and `pantheon.yml`. Both configuration files utilize the same set of properties that define how your Pantheon environments will behave.
 
-Hook into platform workflows and add advanced site configuration via the `pantheon.yml` file. To get started, create a new file named `pantheon.yml` in the root of your site's code repository. Refer to [example.pantheon.yml](https://github.com/pantheon-systems/quicksilver-examples/blob/master/example.pantheon.yml) for example configurations. Refer to the [Pantheon Cloud Integration Examples](https://github.com/pantheon-systems/quicksilver-examples) repository for example scripts for use with Quicksilver Platform Hooks.
+Default values for these properties are set in a `pantheon.upstream.yml` file in an Custom Upstream repository and applied across all sites running that Custom Upstream. The `pantheon.yml` file is used to override the default values set upstream for site-local configurations.
```

Comment:
Instead of the sentence above, maybe use something like:

> When the same configuration value is defined in both files, the value from `pantheon.yml` will override the value from `pantheon.upstream.yml`.